www/source/docs/blueprints.md

```diff
@@ -0,0 +1,66 @@
+# Blueprints
+Blueprints are default implementations of the build phases and runtime hooks for your application. They also provide some default health check hooks where appropriate to ensure your application is functioning reliably. Customized Blueprints can be created to facilitate re-usability of common patterns in your organization for developing, building, and running your applications.
```

Comment:
health checks are just *one* hook that we might provide implementations for. I think it's more appropriate to say "They also provide some default implementations of various life-cycle hooks where appropriate"